docs/_sidebar.md

```diff
@@ -26,6 +26,7 @@
  - [Crash Reporting](/modules/crash)
  - [Database](/modules/database)
  - [Firestore (Beta)](/modules/firestore)
+ - [Dynamic Links](/modules/links)
```

Comment:
Can this be moved above Firestore?  The rest are all in alphabetical order - I'm assuming this was a merge thing because we added it at the same time.

Reply:
Actually, it because I called it Links in the beginning (the name of the library - which in every other place it's after) - but sure, here it has to be before.